packages/node_modules/@ciscospark/react-component-button-controls/src/index.js

```diff
@@ -34,25 +34,31 @@ function ButtonControls(props) {
     showLabels
   } = props;
 
-  const buttonsRendered = buttons.map((button) => {
+  const buttonsRendered = buttons.map((button, idx) => {
     const label = showLabels && button.label ? button.label : '';
+    const ariaLabel = button.accessibilityLabel || button.label;
+    const key = button.key || `button-controls-${idx}`;
 
     if (button.callControl) {
       return (
         <CallControl
-          {...button}
```

Comment:
When we passed all the props as an object, momentum caused react errors, so I manually specified the exact props to pass.

Reply:
we should let them know about this as well I would suggest.

Reply:
It appears as if this is [by design](https://github.com/momentum-design/momentum-ui/blob/master/react/src/lib/Button/index.js#L180) but we can let them know for sure.